polaris/viz/planar.py

```diff
@@ -36,27 +36,41 @@ def plot_horiz_field(ds, ds_mesh, field_name, out_file_name=None,  # noqa: C901
     title: str, optional
         The title of the plot
 
-    vmin, vmax : float, optional
-        The minimum and maximum values for the colorbar
+    vmin : float, optional
+        The minimum values for the colorbar
+
+    vmax : float, optional
+        The maximum values for the colorbar
 
     show_patch_edges : boolean, optional
         If true, patches will be plotted with visible edges
 
-    t_index, z_index: int, optional
-        The indices of 'Time' and 'nVertLevels' axes to select for plotting
-        The default time index is 0 (initial time)
+    t_index: int, optional
+        The indices of ``Time`` axes to select for plotting. The default is 0
+        (initial time)
+
+    z_index: int, optional
+        The indices of ``nVertLevels`` axes to select for plotting. The default
+        is 0 (top level)
 
     cmap : Colormap or str, optional
         A color map to plot
 
     cmap_set_under : str or None, optional
         A color for low out-of-range values
 
+    cmap_set_over : str or None, optional
+        A color for upper out-of-range values
+
     cmap_scale : {'log', 'linear'}, optional
         Whether the colormap is logarithmic or linear
 
     cmap_title : str
         Title for color bar
+
+    figsize : tuple
+        The width and height of the figure in inches. Default is determined
+        based on the aspect ratio of the domain.
```

Comment:
The default approach to determining the figure size didn't work well for the long, skinny aspect ratio of ISOMIP+ plots.